src/man/sssd-ldap.5.xml

```diff
@@ -37,7 +37,7 @@
         <para>
             LDAP back end supports id, auth, access and chpass providers. If you want
             to authenticate against an LDAP server either TLS/SSL or LDAPS
-            is required. <command>sssd</command> <emphasis>does
+            is required. <command>sssd</command> <emphasis> does
```

Comment:
This should not be needed. It renders correctly on my machine.

Reply:
It is as stated above.
I would request a PR that removed this patch.